getting-started/consume-services.hbs.md

```diff
@@ -1,38 +1,27 @@
 
 ## <a id="you-will"></a>What you will do
 
-- Inspect the claim created for the service instance by the application operator.
-- Bind the application workload to the claim so the workload utilizes the service instance.
+- Discover the range of services available to you
+- Create a claim for an instance of one of the services
+- Create two application workloads and bind them to the claim so that the workloads utilize the service instance
 
 ## <a id="overview"></a>Overview
 
-The following diagram depicts a summary of what this walkthrough covers, including the work of the service and application operators described in [Set up services for consumption by developers](set-up-services.md).
+The following diagram depicts a summary of what this tutorial covers.
 
-![Diagram shows the default namespace and service instances namespace. The default namespace has two application workloads, each connected to a service binding. The service bindings connect to the service instance in the service instances namespace through a claim.](../images/getting-started-stk-1.png)
+![Diagram shows the default namespace and a glimpse at what is happening behind the scenes. The default namespace has two application workloads, each connected to a service binding. The service bindings refer to a single claim, which refers to a service instance.](../images/getting-started-stk-1.png)
 
 Bear the following observations in mind as you work through this guide:
 
-1. There is a clear separation of concerns across the various user roles.
-
-    * Application developers set the life cycle of workloads.
-    * Application operators set the life cycle of claims.
-    * Service operators set the life cycle of service instances.
-    * The life cycle of service bindings is implicitly tied to the life cycle of workloads.
-
-2. ProvisionedService is the contract allowing credentials and connectivity information to flow from the service instance to the class claim, to the resource claim, to the service binding, and ultimately to the application workload. For more information, see [ProvisionedService](https://github.com/servicebinding/spec#provisioned-service) on GitHub.
+1. There are a set of four classes preinstalled on the cluster.
+2. There is no configuration or setup required by Service Operators.
+3. ProvisionedService is the contract allowing credentials and connectivity information to flow from the service instance to the resource claim, to the class claim, to the service binding, and ultimately to the application workload. For more information, see [ProvisionedService](https://github.com/servicebinding/spec#provisioned-service) on GitHub.
```

Comment:
**thought:** I wonder whether this is actually essential for the audience of this particular getting started guide, or whether it potentially adds more confusion as there really isn't any API with the name `ProvisionedService` and the diagram linked above doesn't mention it either.